tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -92,12 +94,26 @@
     required=False,
     help="Pass in a dataset file for parsing instead of stdin.",
 )
+@optgroup.option(
+    "--modality",
+    type=click.Choice(["image", "video"]),
+    default=None,
+    help="Modality of the multimodal requests.",
+)
+@optgroup.option(
+    "--max_input_seq_len_for_multimodal",
```

Comment:
nit: max_input_len_for_multimodal. or just `--max_input_len` default to None and saying it should only be set for mm?